src/diag.c

```diff
@@ -187,6 +187,26 @@ bool loc_invalid_p(const loc_t *loc)
       || loc->file_ref == FILE_INVALID;
 }
 
+bool locs_equal(const loc_t *lhs, const loc_t *rhs)
```

Comment:
`loc_eq` already does this.

Reply:
Sorry, I should have checked. It was stupid mistake.